src/migrations/selenium-javascript.js

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import { run, npm } from '../utils';
+import SDKMigration from './base';
+
+class SeleniumJavaScriptMigration extends SDKMigration {
+  static name = '@percy/selenium-webdriver';
+  static aliases = ['@percy/seleniumjs'];
+  static version = '^1.0.0';
+
+  async upgrade() {
+    if (this.installed && this.aliases.includes(this.installed.name)) {
+      await npm.uninstall(this.installed.name);
+    }
+
+    await npm.install(`${this.name}@${this.version}`);
+  }
+
+  transforms = [{
+    message: 'SDK exports have changed, update imports?',
```

Comment:
Should this message be different since the main thing changing is the package name?